cmd/dcrdata/views/treasury.tmpl

```diff
@@ -38,28 +40,28 @@
             <span class="text-secondary fs13">Received</span>
             <br>
             <span class="lh1rem d-inline-block pt-1 fs18 fs14-decimal font-weight-bold">
-            {{- if .Balance}}
-              {{- $received := add .TotalSpent .Balance}}
+            {{- if or $bal.Added $bal.TBase}}
+              {{- $received := add $bal.Added $bal.TBase}}
               {{- template "decimalParts" (amountAsDecimalParts $received true)}} <span class="text-secondary fs14">DCR</span>
             {{- else}}
               <span class="fs18">0</span> <span class="text-secondary fs14">DCR</span>
             {{- end}}
             </span>
             <br>
-            {{/* <span class="text-secondary fs16 lh1rem">{{intComma (add .Balance.NumSpent .Balance.NumUnspent)}} outputs</span> */}}
+            {{/* <span class="text-secondary fs16 lh1rem">{{intComma $bal.TBaseCount}} gens, {{intComma $bal.AddCount}} adds</span> */}}
           </div>
           <div class="d-inline-block text-left pr-2 pb-3">
             <span class="text-secondary fs13">Spent</span>
             <br>
             <span class="lh1rem d-inline-block pt-1 fs18 fs14-decimal font-weight-bold">
-            {{- if .Balance}}
-              {{- template "decimalParts" (amountAsDecimalParts .TotalSpent true)}} <span class="text-secondary fs14">DCR</span>
+            {{- if $bal.Spent}}
+              {{- template "decimalParts" (amountAsDecimalParts $bal.Spent true)}} <span class="text-secondary fs14">DCR</span>
             {{- else}}
               <span class="fs18">0</span> <span class="text-secondary fs14">DCR</span>
             {{- end}}
             </span>
             <br>
-            {{/*  <span class="text-secondary fs16 lh1rem">{{intComma .Balance.NumSpent}} inputs</span> */}}
+            {{/*  <span class="text-secondary fs16 lh1rem">{{intComma $bal.SpendCount}} inputs</span> */}}
```

Comment:
A few of these still have the `/* */` that I added while hacking.  Not sure if you wanted to uncomment any of them.